### PR TITLE
[14.0][MIG]fieldservice_project

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

File: fieldservice_project/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2019 - TODAY, Patrick Wilson
+# Copyright (C) 2021 - TODAY, Patrick Wilson
```
Comment (fshah-initos) [inline fieldservice_project/__init__.py]: You don't need to change the license year

**Review by fshah-initos — APPROVED** [review]
Code review and Functional test. LGTM

**Comment by fshah-initos** [comment]
Hi @Freni-OSI Can please have a look at above comments?

**Comment by fshah-initos** [comment]
It would be great if somone could add reviews and merge the PR.

**Comment by brian10048** [comment]
@Freni-OSI we have since merged this to v13 which has increased tests coverage and also includes a minor bug fix.
Can you please rebase your migration to reflect these v13 improvements?

See #772 and #776

**Comment by fshah-initos** [comment]
Hi @Freni-OSI It would be great if you could look at above comments.

**Comment by Freni-OSI** [comment]
> Hi @Freni-OSI It would be great if you could look at above comments.

for sure. Thanks for the update.

**Comment by brian10048** [comment]
@Freni-OSI can you rebase your migration so that the commit history from v13 remains?  It looks like you included the changes to the code, but the commits are not present

**Comment by brian10048** [comment]
Thank you @Freni-OSI!

**Review by brian10048 — APPROVED** [review]

**Comment by fshah-initos** [comment]
It would be greate if the maintainers could merge the PR ASAP.

**Comment by fshah-initos** [comment]
It seems that PR has approved label maintainer please merge this PR.

**Pull request merged**